File: push-openapi-net-demo/UnbindTag.cs
using Aliyun.Acs.Core;
using Aliyun.Acs.Core.Exceptions;
using Aliyun.Acs.Core.Profile;
using Aliyun.Acs.Push.Model.V20150827;
using System;

namespace AlibabaCloud
{
    class UnbindTag
    {
        static void Main()
        {
            IClientProfile clientProfile = DefaultProfile.GetProfile("cn-hangzhou", "<your access key id>", "<your access key secret>");
            DefaultAcsClient client = new DefaultAcsClient(clientProfile);
            UnbindTagRequest request = new UnbindTagRequest();
            request.AppKey = <Your AppKey>;
            request.KeyType = 1;//1 : device 2 : account 3 : alias
            request.ClientKey = <Your DeviceId>";
            request.TagName = "tag1";

            try
            {
                UnbindTagResponse response = client.GetAcsResponse(request);
                Console.WriteLine("RequestId:" + response.RequestId);
                Console.ReadLine();
            }
            catch (ServerException e)
            {
                Console.WriteLine(e.ErrorCode);
                Console.WriteLine(e.ErrorMessage);
                Console.ReadLine();
            }
            catch (ClientException e)
            {
                Console.WriteLine(e.ErrorCode);
                Console.WriteLine(e.ErrorMessage);
                Console.ReadLine();
            }
        }

    }
}

File: OpenApi2.0/push-openapi-nodejs-demo/UnbindTag.js
var ALY = require('aliyun-sdk');

var push = new ALY.PUSH({
      accessKeyId: '<your access key id>',
      secretAccessKey: '<your access key secret>',
      endpoint: 'http://cloudpush.aliyuncs.com',
      apiVersion: '2016-08-01'
    }
);

push.unbindTag({
    AppKey: '<your Appkey>',
    KeyType : "DEVICE"//DEVICE：是设备，ACCOUNT：是账号，ALIAS：是别名
    ClientKey : '<your deviceId or your account>',
    TagName : 'tag3'
  }, function (err, res) {
    console.log(err, res);
  });

return ;
